SearchQnA/README_miscellaneous.md

```diff
@@ -0,0 +1,47 @@
+# SearchQnA Docker Image Build
+
+## Table of Contents
+
+1. [Build MegaService Docker Image](#build-megaservice-docker-image)
+2. [Build UI Docker Image](#build-ui-docker-image)
+3. [Generate a HuggingFace Access Token](#generate-a-huggingface-access-token)
+4. [Troubleshooting](#troubleshooting)
+
+## Build MegaService Docker Image
+
+To construct the Megaservice of SearchQnA, the [GenAIExamples](https://github.com/opea-project/GenAIExamples.git) repository is utilized. Build Megaservice Docker image via command below:
```

Comment:
"Build Megaservice Docker image via command below" - Why are we using the word "via"? A clearer sentence would be "Use the following command to build the Megaservice Docker image:"
.
Please look at the use of the word "via" throughout this documentation.